python/ray/train/v2/api/train_fn_utils.py

```diff
@@ -81,6 +81,9 @@ def train_func(config):
     Args:
         metrics: The metrics you want to report.
         checkpoint: The optional checkpoint you want to report.
+        checkpoint_dir_name: The optional directory to store the checkpoint.
+            If not set, the checkpoint will be stored in the default storage path.
+            If set, make sure this value is unique for each iteration.
```

Comment:
```suggestion
        checkpoint_dir_name: Custom name for the checkpoint directory. If not provided, a unique directory
    name will be automatically generated. If provided, it must be unique
    across all checkpoints to avoid naming collisions. Consider including identifiers
    such as the epoch or batch index in the name.
```

Reply:
nit: 

> If provided, it must be unique
	    across all checkpoints to avoid naming collisions.

For this, maybe specify that this is referring to different calls to `report` within a worker? As opposed to having diff checkpoint names from diff workers.

Reply:
I just added `per worker` to @justinvyu 's text - lmk if this clears it up.